src/util/Layout.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Text } from '@ui5/webcomponents-react/lib/Text';
+
+export const TextWrapper = ({ children, isOnHeader, style }) => {
```

Comment:
I added this first `TextWrapper` as a "util" component, but I am not sure if this is the optimal approach.
Feedback is welcome! 😄

Reply:
yeah. I was not a fan of this wrappers guys in our project honestly.
i think you could remove it.
apart from the wrapping, everything would be the same